src/App.tsx

```diff
@@ -1,17 +1,15 @@
-import { Outlet } from 'react-router-dom';
-import { Header } from './components/Header';
+// import { Outlet } from 'react-router-dom';
+import { Header } from 'components/Header';
+import { HomePage } from './pages/HomePage';
 import './index.scss';
 
 export const App = () => {
   return (
     <html lang="en" className="page">
```

Comment:
i dont think we need an html here?
we put all of our code from App to body in our index.html
![image](https://github.com/fe-jul23-team6/product_catalog/assets/108063043/01322359-b306-4a8e-8d19-1d9bc39e8031)
and we have the HTML tag there as well as body, so we should remove those from here